extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/serde/tableExtractors.scala

```diff
@@ -144,9 +146,12 @@ class ResolvedTableTableExtractor extends TableExtractor {
  */
 class IdentifierTableExtractor extends TableExtractor {
   override def apply(spark: SparkSession, v1: AnyRef): Option[Table] = {
-    val namespace = invokeAs[Array[String]](v1, "namespace")
-    val table = invokeAs[String](v1, "name")
-    Some(Table(None, Some(quote(namespace)), table, None))
+    val identifier = v1.asInstanceOf[Identifier]
+    if (PathIdentifier.isPathIdentifier(identifier)) {
```

Comment:
Let's make it simple here. Only check runSQLonFile && new URI(table).isAbsolute

Reply:
What is the reason for removing `hasDeltaNamespace(namespace)`?

Reply:
path-based table is not deltalake-specific